### DISCUSSION TITLE
Frida's system requirement

### DISCUSSION THREAD

**Discussion by hecodeit** [cmubig/Frida#7] [Q&A]
Does somebody know about Frida's system requirement? If it's working with computer like Raspberry Pi?

**Comment by pschaldenbrand** [comment]
Sorry I will update the documentation soon with some more info on system requirements. It uses a lot of machine learning on PyTorch, so it's recommended to have a decent NVIDIA GPU like a 3080 or 3090 or better.

**Reply by hecodeit** [reply]
Reading the paper and code these days, and thinking if it's there a version of "Frida-Mini"? 

Just Real2Sim2Real function, non advanced function like "natural language and style images". So preprocess the input like "an image, an image with style, text to image result" by users. User choose to using Stable Diffusion, Midjourney,  DALL·E, etc. 

So will make "Frida-Mini" work with the more system without high end GPU?

**Reply by pschaldenbrand** [reply]
FRIDA's core functionality is based on optimization and ML, so I think a GPU will always be necessary, even with reduced features. But I've run it on a laptop with only 4gb of GPU memory before. I'd recommend around 8Gb+ of vRAM for FRIDA though.

**Reply by hecodeit** [reply]
My laptop run with RTX 3060 6GB GPU memory. Windows 11 system. I try install Frida with WSL Ubuntu 20.04.6 but no success. Will try later, find out where is the problem.

**Reply by pschaldenbrand** [reply]
I've tried to get pytorch with cuda on WSL Ubuntu before and it was very challenging. Looks like NVIDIA has some new directions here: https://docs.nvidia.com/cuda/wsl-user-guide/index.html
